lib/Driver/Driver.cpp

```diff
@@ -1340,22 +1340,34 @@ void Driver::buildOutputInfo(const ToolChain &TC, const DerivedArgList &Args,
 
   if (const Arg *A = Args.getLastArg(options::OPT_g_Group)) {
     if (A->getOption().matches(options::OPT_g))
-      OI.DebugInfoKind = IRGenDebugInfoKind::Normal;
+      OI.DwarfInfoKind = IRGenDwarfDebugInfoKind::Normal;
     else if (A->getOption().matches(options::OPT_gline_tables_only))
-      OI.DebugInfoKind = IRGenDebugInfoKind::LineTables;
+      OI.DwarfInfoKind = IRGenDwarfDebugInfoKind::LineTables;
     else if (A->getOption().matches(options::OPT_gdwarf_types))
-      OI.DebugInfoKind = IRGenDebugInfoKind::DwarfTypes;
+      OI.DwarfInfoKind = IRGenDwarfDebugInfoKind::DwarfTypes;
     else
       assert(A->getOption().matches(options::OPT_gnone) &&
              "unknown -g<kind> option");
   }
 
+  if (const Arg *A = Args.getLastArg(options::OPT_debug_info_format)) {
+    if (A->containsValue("dwarf"))
```

Comment:
Nitpick: Since this isn't a multi-valued option, comparing against `getValue` is probably clearer.